packages/apps/spaces/app/src/components/EmailMultiCreatableSelect/MultiValueWithActionMenu.tsx

```diff
@@ -117,28 +125,98 @@ export const MultiValueWithActionMenu: FC<MultiValueWithActionMenuProps> = ({
     );
   };
 
+  if (showEditInput) {
+    const handleChangeValue: ReactEventHandler<HTMLElement> = (event) => {
+      const newValue = value.map((e: SelectOption<string>) =>
+        e.value === rest?.data?.value
+          ? {
+              label: (event?.target as HTMLInputElement)?.value,
+              value: (event?.target as HTMLInputElement)?.value,
+            }
+          : e,
+      );
+      onChange(newValue);
+      setEditInput(false);
+    };
+
+    return (
+      <Tooltip label={validationMessage ? validationMessage : ''}>
+        <Input
+          autoFocus
+          className='w-auto inline text-warning-700 h-8 hover:border-transparent focus:border-transparent'
+          onBlur={(e) => {
+            handleChangeValue(e);
+          }}
+          onKeyDown={(e) => {
+            e.stopPropagation();
+            if (e.key === 'Enter') {
+              handleChangeValue(e);
+            }
+          }}
+          defaultValue={rest?.data?.value}
+        />
+      </Tooltip>
+    );
+  }
+
   return (
     <Menu>
-      <MenuButton
-        sx={{
-          '&[aria-expanded="true"] > span > span': {
-            bg: isContactWithoutEmail
-              ? 'warning.50 !important'
-              : 'primary.50 !important',
-            color: isContactWithoutEmail
-              ? 'warning.700 !important'
-              : 'primary.700 !important',
-            borderColor: isContactWithoutEmail
-              ? 'warning.200 !important'
-              : 'primary.200 !important',
-          },
-        }}
-      >
-        <chakraComponents.MultiValue {...rest}>
-          {rest.children}
-        </chakraComponents.MultiValue>
-      </MenuButton>
+      <Tooltip label={validationMessage ? validationMessage : ''}>
+        <MenuButton
```

Comment:
just a suggestion: we have in place the new Menu component in the future you can use the new **component**

Reply:
The migration of this specific component needs to be done separately as simply swapping this one for the new one created breaks the app